collector/src/rustc-fake.rs

```diff
@@ -467,8 +467,9 @@ fn process_self_profile_output(prof_out_dir: PathBuf, args: &[OsString]) {
 #[cfg(windows)]
 fn exec(cmd: &mut Command) {
```

Comment:
FWIW using return type `-> !` (like the Unix version) would have caught this

Reply:
Ahh I forgot that part when I posted @mystor's suggestion (https://github.com/rust-lang/rustc-perf/pull/1366#discussion_r928057956).